README.md

```diff
@@ -42,6 +42,117 @@ val df = sqlContext.bigQuerySelect(
 df.saveAsBigQueryTable("my-project:my_dataset.my_table")
 ```
 
+# Why this repo?
+
+This repository was created because the parent project `sportify/bigquery` is no longer being actively developed.
```

Comment:
You have `portify/bigquery`. Should be `spark-bigquery`

Reply:
@jakeorr the original project was `sportify/spark-bigquery`